micromamba/tests/channel_a/linux-64/repodata.json

```diff
@@ -35,7 +37,7 @@
       "build": "abc",
       "build_number": 0,
       "depends": [
-        "a"
+        "a", "__glibc=2.31"
```

Comment:
Why do we need to add this?

Reply:
I have no idea why, I did not even touch that file, it is so strange that this json file was modified

Reply:
This file (among other ones) is generated when running the Python tests, we should probably ensure that it is cleanup when the tests are done.